core/lib/globals_shared.h

```diff
@@ -1870,10 +1870,16 @@ typedef union _dr_simd_t {
 } dr_simd_t;
 #    endif
 #    ifdef X64
+#        define MCXT_NUM_SIMD_SSE_AVX_SLOTS \
+            32 /**< Unused in ARM/AArch64   \
```

Comment:
Missing trailing `.`.  Several below as well.

Reply:
I've removed the define from ARM.